frontend/hydrogen.config.json

```diff
@@ -363,7 +363,7 @@
         },
         {
           "key": "rounded",
-          "radius": "5px"
+          "radius": "6px"
```

Comment:
As mentioned, this changes the `rounded` key sitewide, so I'd recommend against it for now.

Reply:
So, I did some testing today between `5px` and `6px` and in this specific use case, I think `6px` looks a bit better since they match the outer radius slightly better. I think this is because it is doubling the `3px` of the border that is has.

<img width="864" alt="Screen Shot 2022-10-05 at 2 31 03 PM" src="https://user-images.githubusercontent.com/4127998/194135874-74c7d43e-65be-407f-8014-dc9dcaca1456.png">

Reply:
Here's the extent of the differences that show up in storybook when changing "rounded" to a divisible number. Doesn't seem to be too sweeping?
https://www.chromatic.com/pullrequest?appId=61e099a1bb1465003a73d3ce&number=4114&view=changes

Reply:
Okiedoke, looks good to me then!